joboard/urls.py

```diff
@@ -14,4 +14,5 @@
     path('signup/', user_views.signup_view, name='signup'),
     path('login/', auth_views.LoginView.as_view(template_name='signup/login.html'), name='login'),
     path('logout/', auth_views.LogoutView),
+    url(r'^jobs/', include('searchJob.urls')),
```

Comment:
sorry i lost you,
Why do you have two urls?
joboard/urls
searchjob/urls
Is it 2 applications?

Reply:
> sorry i lost you,
> Why do you have two urls?
> joboard/urls
> searchjob/urls
> Is it 2 applications?

Yes, we have 3 apps that every one of them on separate directory and work as an app